.github/workflows/ci.yml

```diff
@@ -68,7 +68,15 @@ jobs:
         run: python -m pip install .[develop]
       - name: "Run tests"
         run: hatch -v -e it run test
+        timeout-minutes: 120
         env:
           # elastic/endpoint fetches assets from GitHub, authenticate to avoid
           # being rate limited
           ASSETS_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            +      # Artifact will show up under "Artifacts" in the "Summary" page of runs
```

Comment:
Please fix indentation and remove `+`.

Reply:
Thanks, fixed in [`b7286bd` (#416)](https://github.com/elastic/rally-tracks/pull/416/commits/b7286bd49c2800b97401c7b069bcc3c5540582c5)

Reply:
Thank you.